html/index.html

```diff
@@ -14,6 +14,15 @@
   <script src="app.js"></script>
 </head>
 <body>
-<div class="loader">Lade<br /><span class="spinner"></span><br />Karten &amp; Knoten...</div>
+<div class="loader">
+  <p>
+    Lade<br />
+    <span class="spinner"></span><br />
```

Comment:
spam (strong em usw.) sind fuer mich inline elemente. Hab jede Sache in eine Zeile - vorallem um mergekonflikte bei Anpassungen zu vermeiden.